platform/default/include/mbgl/gfx/headless_frontend.hpp

```diff
@@ -48,7 +54,7 @@ class HeadlessFrontend : public RendererFrontend {
     LatLng latLngForPixel(const ScreenCoordinate&);
 
     PremultipliedImage readStillImage();
```

Comment:
shall `readStillImage()` return `RenderResult` too? (Might be added later)

Reply:
This is an interesting function as it has to be called from the rendering thread. It can be used together with the `Map::renderStill(...)` to fetch the still image in the callback.